scripts/globals/magic.lua

```diff
@@ -603,7 +613,12 @@ end;
 function handleAfflatusMisery(caster, spell, dmg)
     if (caster:hasStatusEffect(dsp.effect.AFFLATUS_MISERY)) then
         local misery = caster:getMod(dsp.mod.AFFLATUS_MISERY);
-        local miseryMax = caster:getMaxHP() / 4;
+        -- According to BGWiki Caps at 300 magic damage.
+        local miseryMax = 300
+
+        if caster:getMainJob() == dsp.job.WHM and caster:getMainLvl() >= 75 then
```

Comment:
Remove main job and main level check. They are not needed. getMerit handles this.